dotnet/src/Connectors/Connectors.Memory.AzureAISearch/AzureAISearchMemoryRecordServiceOptions.cs

```diff
@@ -0,0 +1,36 @@
+﻿// Copyright (c) Microsoft. All rights reserved.
+
+using System.Text.Json.Nodes;
+using Microsoft.SemanticKernel.Memory;
```

Comment:
Small reminder - if we will use `Vector/VectorDb` naming instead of `Memory`, we should probably do the same renaming for this namespace as well.

Reply:
Thanks, I've added it to my list to figure out a good name.